components/tabs/demo/tabs.html

```diff
@@ -73,30 +79,32 @@ <h3>Tabs (responsive)</h3>
 
 			<d2l-demo-snippet>
 				<template>
-					<d2l-tabs>
-						<d2l-tab-panel text="S18">Tab content for S18</d2l-tab-panel>
-						<d2l-tab-panel text="W18">Tab content for W18</d2l-tab-panel>
-						<d2l-tab-panel text="F17">Tab content for F17</d2l-tab-panel>
-						<d2l-tab-panel text="S17">Tab content for S17</d2l-tab-panel>
-						<d2l-tab-panel text="W17">Tab content for W17</d2l-tab-panel>
-						<d2l-tab-panel text="F16">Tab content for F16</d2l-tab-panel>
-						<d2l-tab-panel text="S16">Tab content for S16</d2l-tab-panel>
-						<d2l-tab-panel text="W16">Tab content for W16</d2l-tab-panel>
-						<d2l-tab-panel text="F15">Tab content for F15</d2l-tab-panel>
-						<d2l-tab-panel text="S15">Tab content for S15</d2l-tab-panel>
-						<d2l-tab-panel text="W15">Tab content for W15</d2l-tab-panel>
-						<d2l-tab-panel text="F14">Tab content for F14</d2l-tab-panel>
-						<d2l-tab-panel text="S14">Tab content for S14</d2l-tab-panel>
-						<d2l-tab-panel text="W14">Tab content for W14</d2l-tab-panel>
-						<d2l-tab-panel text="F13">Tab content for F13</d2l-tab-panel>
-						<d2l-tab-panel text="S13">Tab content for S13</d2l-tab-panel>
-						<d2l-tab-panel text="W13">Tab content for W13</d2l-tab-panel>
-						<d2l-tab-panel text="F12">Tab content for F12</d2l-tab-panel>
-						<d2l-tab-panel text="S12">Tab content for S12</d2l-tab-panel>
-						<d2l-tab-panel text="W12">Tab content for W12</d2l-tab-panel>
-						<d2l-tab-panel text="F11">Tab content for F11</d2l-tab-panel>
-						<d2l-tab-panel text="S11">Tab content for S11</d2l-tab-panel>
-					</d2l-tabs>
+					<div style="max-width: 500px;">
```

Comment:
This demo was broken before, as it wasn't actually responsive. The tabs container just ended up overflowing the demo snippet container.